### PR TITLE
Switch movie pages

### PR DESCRIPTION
**Pull request by danielmolnar**
- Implemented ability to fetch next page of movies, since the API has a request limit of 20 movies at a time
- Fixed a bug which crashed the page sometimes when there is no user profile set
- Implemented more fallback logic when the API doesnt provide movie posters

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -136,6 +150,11 @@ function App() {
     (list) => list.favorites.length !== 0
   );
 
+  const nextHandler = () => {
```
Comment (KarinLiegmann) [inline client/src/App.js:153]: I would probably give this function a more "speaking" name like "handleNextPage"

File: server/src/server.js [hunk 2]
```diff
@@ -35,9 +35,7 @@ server.get('/users/:userId', (req, res) => {
   User.findById(userId)
     .then((user) => res.json(user))
     .catch((error) => {
-      console.log(
-        'User not yet registered. Please consider creating a profile'
-      );
+      console.error(error.message);
```
Comment (KarinLiegmann) [inline server/src/server.js:38]: Good job. :) Much better than console-logs

**Review by KarinLiegmann — CHANGES_REQUESTED** [review]
Hey Daniel, looks good to me! I only had one small comment regarding a function-name. But otherwise great!

File: client/src/App.js (hunk 1 above)
Comment (danielmolnar) [reply client/src/App.js:153]: Good catch!

File: server/src/server.js (hunk 2 above)
Comment (danielmolnar) [reply server/src/server.js:38]: True that :)

**Pull request merged**